app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,40 @@
+class ReviewsController < SecuredController
+  skip_before_action :authorize_request, only: %i[index show]
+
+  # GET /reviews
+  def index
+    user_sub = request.headers['body']
+    fab = User.find_by(user_sub: user_sub)
+    review = fab.reviews
+    render json: review
+  rescue ActiveRecord::RecordNotFound
+    head :not_found
+  end
+
+  # GET /reviews/1
+  def show
+    review = Review.where(laptop_id: params[:id])
+    render json: review
+  rescue ActiveRecord::RecordNotFound
+    head :not_found
+  end
+
+  # POST /reviews
+  def create
+    fab = User.find_by(user_sub: params[:user_sub])
+    review = Review.create!(user_email: params[:user_email],
```

Comment:
Probably we forgot to sanitize our parameters here? This opens the API to SQLi attacks